tools/actions/merge.sh

```diff
@@ -39,24 +39,18 @@ git log -1 HEAD^ --pretty='format:%B' | git interpret-trailers --parse --no-divi
 commit_title=$(git log -1 --pretty='format:%s')
 commit_body=$(git log -1 --pretty='format:%b')
 
-jq -n \
-    --arg title "${commit_title}" \
-    --arg body "${commit_body}" \
-    --arg head "${commit_head}" \
-    '{merge_method:"squash",commit_title:$title,commit_message:$body,sha:$head}' > output.json
-cat output.json
-if ! gh api -X PUT "repos/${OWNER}/${REPOSITORY}/pulls/${pr}/merge" --input output.json > output; then
+if ! gh pr merge "$pr" --squash --body "$commit_body" --subject "$commit_title" --match-head-commit "$commit_head" --admin > output; then
```

Comment:
Right, although I'm quite sure we don't need to keep it, we can pass a `--jq` flag
```suggestion
if ! commits="$(gh pr merge "$pr" --squash --body "$commit_body" --subject "$commit_title" --match-head-commit "$commit_head" --admin --jq 'if .merged then .sha else halt_error end')"; then
```

Reply:
Hum so I actually try that to land https://github.com/nodejs/node/pull/57023, and it looks like `gh pr merge` has no output :/ Which means we cannot get the SHA of the resulting commit, I've opened https://github.com/cli/cli/issues/10746 to request its addition.